bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/DY01jToEE_MEE-0p1To4_LO_5f/DYto2E-1Jets_MEE-0p1To4_madgraphMLM-pythia8_proc_card.dat

```diff
@@ -0,0 +1,6 @@
+import model sm-ckm_no_b_mass
+
+generate p p > e+ e-  @0
+add process p p > e+ e- j  @1
+
+output DYto2E-1Jets_MEE-0p1To4_madgraphMLM-pythia8 -nojpeg
```

Comment:
What's the right naming convention, `1Jets` or `1Jet` ? (or none of the above..)

Reply:
We don't have a strict policy for this, but you should use the same name as in the `*proc_card.dat`.
I would suggest to rename the proc and run cards as `DY01j_LO_5f_proc_card.dat` and `DY01j_LO_5f_run_card.dat`.
And then change the last line to:
`output DY01j_LO_5f -nojpeg`